packages/embark-specialconfigs/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "embark-specialconfigs",
+  "version": "4.0.0",
```

Comment:
As of https://github.com/embark-framework/embark/tree/4.0.x this version number is obsolete.

We need to get in the 4.0.x commits first..

Reply:
See also: https://github.com/embark-framework/embark/pull/1470#discussion_r269660707. That's for a different PR, but same concern applies to this one.